src/js/components/deployments/deployment-wizard/review.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import pluralize from 'pluralize';
+
+import { Chip, List } from '@material-ui/core';
+import { InfoOutlined as InfoOutlinedIcon } from '@material-ui/icons';
+
+import ExpandableDeviceAttribute from '../../devices/expandable-device-attribute';
+import { getRemainderPercent } from '../../../helpers';
+
+const Review = props => {
+  const { deploymentDeviceIds, device, group, isEnterprise, phases, release } = props;
+
+  // Create 'phases' for view only
+  var deploymentPhases = phases ? phases : [{batch_size: 100}];
+  const start_time = deploymentPhases[0].start_ts || new Date().toISOString();
```

Comment:
...and replace the `toISOString()` with `.toLocaleString(undefined, dateOptions)` here